Std/Tactic/Classical.lean

```diff
@@ -24,6 +24,19 @@ Consider using `classical` instead if you want to use the decidable instance whe
 macro (name := classical!) "classical!" : tactic =>
   `(tactic| have em := Classical.propDecidable)
 
+/--
+`classical t` runs `t` in a scope where `Classical.propDecidable` is a low priority
+local instance.
+-/
+def classical [Monad m] [MonadEnv m] [MonadFinally m] [MonadLiftT MetaM m] (t : m α) :
+    m α := do
+  modifyEnv Meta.instanceExtension.pushScope
+  Meta.addInstance ``Classical.propDecidable .local 10
```

Comment:
Maybe the priority should be an optional parameter of some kind?

Reply:
Let's not over-engineer.